### DISCUSSION TITLE
Mastercard (MPGS) Native Payment Gateway

### DISCUSSION THREAD

**Discussion by sfoulad** [inspireui/support#5681] [New Feature/Ideas Requests]
Add support to mastercard payment gateway (MPGS) 

https://developer.mastercard.com/product/payment-gateway-services-mpgs

**Comment by sfoulad** [comment]
Mastercard is a main payment gateway in many countries

**Comment by AkimotoKazue** [comment]
Great ;) https://github.com/Mastercard-Gateway/woocommerce-simplify-payment-gateway-plugin 
https://github.com/Mastercard-Gateway/gateway-woocommerce-module

**Comment by Yan-Aleman** [comment]
Is someone already using this gateway?

**Comment by sfoulad** [comment]
Hopefully it will be added as native gateway on the app

**Reply by Yan-Aleman** [reply]
What plugin are you using on the site?

**Comment by Yan-Aleman** [comment]
It would be very cool if InspireUi develops its own acquiring based on MasterCard Mpgs, with support for national currencies of different countries and split payments for marketplaces (Wcfm).  Even with a higher percentage per transaction than other gateways.  This integration would be much better!

**Comment by sfoulad** [comment]
Mpgs have native integration will be amazing if it can be added
